lib/krane/kubernetes_resource.rb

```diff
@@ -409,11 +411,26 @@ def self.go_template_for(kind, name)
       def self.extract_all_from_go_template_blob(blob)
         blob.split(EVENT_SEPARATOR).map do |event_blob|
           pieces = event_blob.split(FIELD_SEPARATOR, FIELDS.length)
+
+          series = pieces[FIELDS.index(".series")]
+          last_timestamp = pieces[FIELDS.index(".lastTimestamp")]
+
+          # Find the right count and last_timestamp according to Events API version
+          count, last_timestamp = if series.present? && series != "<no value>" # Events/v1
+            count_regex = /count:(?<value>\S+?(?=\s))/
+            lot_regex = /lastObservedTime:(?<value>\S+?(?=\]))/
+            [count_regex.match(series)['value'], lot_regex.match(series)['value']]
+          elsif last_timestamp != "<no value>" # Events/v1beta1
+            [pieces[FIELDS.index(".count")], last_timestamp]
+          else # Fallback to eventTime
+            [1, pieces[FIELDS.index(".eventTime")]]
```

Comment:
What does this case cover?

Reply:
I was trying to cover a case where `lastTimeStamp`, `deprecatedTimeStamp` and `series` where empty.

I will check what happens on `1.11` from `1.16` and will post here.